NIPTool/API/external/api/api_v1/endpoints/login.py

```diff
@@ -13,15 +17,16 @@
 def login_for_access_token(
     form_data: OAuth2PasswordRequestForm = Depends(), config: dict = Depends(temp_get_config)
 ):
-    user: UserInDB = authenticate_user(form_data.username, form_data.password)
+
+    user: User = authenticate_user(form_data.username, form_data.password)
     if not user:
         raise HTTPException(
             status_code=status.HTTP_401_UNAUTHORIZED,
             detail="Incorrect username or password",
             headers={"WWW-Authenticate": "Bearer"},
         )
     access_token_expires = timedelta(minutes=config.get("ACCESS_TOKEN_EXPIRE_MINUTES"))
```

Comment:
```suggestion
    access_token_expires: datetime.timedelta = datetime.timedelta(minutes=config.get("ACCESS_TOKEN_EXPIRE_MINUTES"))
```
suggest to change so that it is more clear what the type is coming from